### DISCUSSION TITLE
Is it possible to get walker that travelled the furthest in 3D ?

### DISCUSSION THREAD

**Discussion by igennart** [PMEAL/pytrax#20] [Q&A]
Dear, 

Is it possible to get the walker that travelled the furthest in 3D, the same way we can see it in 2D ?
And is it possible to get (for 2D and 3D) the biggest displacement length ?

Thanks,

**Comment by jgostick** [comment]
The `rw` object, is an instance of the `RandomWalk` class, and it has numerous attributes, including ``msd``.  It's been a while since I used this package, and I haven't checked, but I "think" this array has the msd for each walk at each step, so you can just look for the maximum value there.
